sklearn/linear_model/logistic.py

```diff
@@ -1494,6 +1484,16 @@ class LogisticRegressionCV(LogisticRegression, BaseEstimator,
         To lessen the effect of regularization on synthetic feature weight
         (and therefore on the intercept) intercept_scaling has to be increased.
 
+    multi_class : str, {'ovr', 'multinomial'}
+        Multiclass option can be either 'ovr' or 'multinomial'. If the option
+        chosen is 'ovr', then a binary problem is fit for each label. Else
+        the loss minimised is the multinomial loss fit across
+        the entire probability distribution. Works only for the 'newton-cg',
+        'sag' and 'lbfgs' solver.
```

Comment:
And the "saga" solver as well. The error message of the `ValueError` for then input check at line ~429 also need a similar update.